src/App.js

```diff
@@ -1,34 +1,58 @@
-import logo from './logo.svg';
 import './App.css';
-import {Fragment, useEffect} from "react";
-import checkboxEmpty from "./assets/checkboxEmpty.png";
-import checkboxFull from "./assets/checkboxFull.png";
-import trashcan from "./assets/trashcan.png";
+import { useEffect} from "react";
 import Tools from "./Tools";
 import Tasks from "./Tasks";
 import AddTask from "./AddTask";
 import {useState} from "react";
 import DeleteTasks from "./DeleteTasks";
-import Task from "./Task";
 import EditTask from "./EditTask";
-import DeleteModal from "./DeleteModal";
+import SortBy from "./SortBy";
 
+import { initializeApp } from "firebase/app";
+import { getFirestore, query, orderBy, collection, doc, updateDoc, deleteDoc, setDoc, serverTimestamp } from "firebase/firestore";
+import {useCollectionData} from "react-firebase-hooks/firestore";
+import {generateUniqueID} from "web-vitals/dist/modules/lib/generateUniqueID";
 
 
+const firebaseConfig = {
+    apiKey: "AIzaSyDOnElC_vlc65Fl1WVTUVg5pMcOivIv-Cw",
+    authDomain: "cs124-lab3-a5cf8.firebaseapp.com",
+    projectId: "cs124-lab3-a5cf8",
+    storageBucket: "cs124-lab3-a5cf8.appspot.com",
+    messagingSenderId: "1089016344467",
+    appId: "1:1089016344467:web:a503e3b598a5e467acc266"
+};
+
+const firebaseApp = initializeApp(firebaseConfig);
+const db = getFirestore(firebaseApp);
+
+const collectionName = "tasks";
+
 function App(props) {
-    const [tasks, setTasks] = useState(props.initialData);
     const [showCompleted, setShowCompleted] = useState(true);
-    const [nextID, setNextID] = useState(10); //TODO - fix initial state
     const [selectedTaskIDs, setSelectedTaskIDs] = useState([]);
-    // TODO - set appropriately once we figure out nav
     // modes: home, delete, edit
     const [mode, setMode] = useState("home");
 
     const [editTaskID, setEditTaskID] = useState(1);
 
+    const Queries = {
+        CreatedSort: query(collection(db, collectionName), orderBy("created")),
```

Comment:
You're duplicating the collection each time.

What about just mapping to the orderBy instead.

Then, you could create with query(collection(...), sortBy)